README.md

```diff
@@ -55,16 +55,17 @@ This is the common workflow to build and deploy locally:
   make info-swarm
 
   # open front-end in the browser
-  #  localhost:9081 - simcore front-end site
+  #  127.0.0.1.nip.io:9081 - simcore front-end site
   #
-  xdg-open http://localhost:9081/
+  xdg-open http://127.0.0.1.nip.io:9081/
 
   # stops
   make down
 ```
 
-Services are deployed in two stacks:``simcore-stack`` comprises all core-services in the framework
-and ``ops-stack`` is a subset of services from [ITISFoundation/osparc-ops](https://github.com/ITISFoundation/osparc-ops) used
+Some routes can only be reached via DNS such as `UUID.services.DNS`. Since `UUID.services.127.0.0.1` is **not a valid DNS**, the solution is to use [nip.io](https://nip.io/). A service that maps ``<anything>[.-]<IP Address>.nip.io`` in "dot", "dash" or "hexadecimal" notation to the corresponding ``<IP Address>``.
```

Comment:
so one question here. how will this look like in production deploys? something like UUID.osparc.io?

Reply:
The service will be exposed on `UUID.services.osparc.io`.

Reply:
Yes, @Surfict should have already setup the required DNS schemes on some of the deployments. Please let me know again what is missing.